Sources/MongoSwift/ReadWriteConcern.swift

```diff
@@ -115,5 +115,134 @@ public class ReadConcern: Equatable, CustomStringConvertible {
         mongoc_read_concern_destroy(readConcern)
         self._readConcern = nil
     }
+}
+
+/// A class to represent a MongoDB write concern.
+public class WriteConcern: Equatable, CustomStringConvertible {
```

Comment:
nit: for debugability would it make sense to make it describable too, and use the `asDocument` representation so you can call `print` on it?